src/components/atoms/Accordion/AccordionUL.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Main = styled.button`
+  background-color: ${({ theme }) => theme.main.colors.purple};
+  color: ${({ theme }) => theme.main.colors.white};
+  border-color: ${({ theme }) => theme.main.colors.purple};
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  width: 720px;
+  flex-direction: row;
```

Comment:
Please remove duplicate styles such as this one

Reply:
removed duplicates